cd/settings.xml

```diff
@@ -0,0 +1,24 @@
+<settings>
```

Comment:
can/should we move this up to the top-level dir?

i don't have any concerns, fwiw.